folding/utils/config.py

```diff
@@ -432,6 +432,12 @@ def add_validator_args(cls, parser):
             "5CQ9KNHy9qvRGhLWeV37agEpmLckSgMXzbZWEEXwbupSCTQy",
         ],
     )
+    parser.add_argument(
+        "--neuron.gjp_address",
+        type=str,
+        help="The IP address and port of the global job pool.",
+        default="184.105.4.4:8030",

```

Comment:
how can we automate this? is this fixed address something that will work for all validators? If it could have some variability, we should indicate how to determine this on update

Reply:
I will think about, nothing from the top of my head